### ISSUE TITLE
Scrolls Upward When Reordering or Resizing Column

### ISSUE THREAD

**Issue by holospice18** [adazzle/react-data-grid#3360]
## Describe the bug 
On Firefox when resizing or reordering column while scroll is further down it scrolls up as if scroll bar's up arrow is clicked.

## To Reproduce 

1. Resizable column like https://adazzle.github.io/react-data-grid/#/animation
    or reordering https://adazzle.github.io/react-data-grid/#/columns-reordering
2. Scroll down 
3. Select any cell
4. Resize or reorder column

## Environment

- `react-data-grid` version: ^7.0.0-beta.36 and ^7.0.0-beta.40
- `react`/`react-dom` version: ^18.2.0
- Windows 10
- Firefox 118.0.2 (64-bit)

## Additional context 
Only happens when grid body is focused then resize or reorder column header

**Comment by DharanBro** [comment]
@holospice18 Is this still happening? I don't see this happening with the current demo page. https://adazzle.github.io/react-data-grid/#/animation

**Comment by holospice18** [comment]
@DharanBro 
No, it was fixed on https://adazzle.github.io/react-data-grid/#/animation
but still persist on https://adazzle.github.io/react-data-grid/#/columns-reordering
